features/help.js

```diff
@@ -99,6 +100,12 @@ async function respondToHelp({ message, client }) {
     user: message.user,
     text: helpMarkdown,
   });
+
+  winston.debug("successfully posted ephemeral help message to Slack", {
```

Comment:
I wonder if there might be a way to hook this kind of logging to the api call itself. (Worst case maybe throw both in a util function, although that might make the code somewhat unreadable.)